src/main/java/org/intellij/sonar/configuration/partials/SonarResourcesTableView.java

```diff
@@ -32,14 +33,18 @@
     @Nullable
     @Override
     public String valueOf(Resource sonarResource) {
-      if (Resource.QUALIFIER_PROJECT.equals(sonarResource.getQualifier())) {
-        return "Project";
-      } else
-        if (Resource.QUALIFIER_MODULE.equals(sonarResource.getQualifier())) {
+      SonarQualifier sonarQualifier = SonarQualifier.valueFrom(sonarResource.getQualifier());
+      if (sonarQualifier == null) {
+        return null;
```

Comment:
why return null here?

Reply:
Hm, good question, I don't remember. Returning the raw qualifier is probably the better choice.